src/components/Form/FetchDocumentForm.jsx

```diff
@@ -54,10 +60,6 @@ const FetchDocumentForm = () => {
     }
   };
 
-  const formRowStyle = {
-    margin: '20px 0'
-  };
-
   /* eslint-disable jsx-a11y/label-has-associated-control */
```

Comment:
I think it's safe enough to turn these off now that our labels are now properly control with FormControl. I've turned removed the linters for this page and it seems to not show any errors even after running the linters manually with npm run lint.